### DISCUSSION TITLE
How to get owner of a token

### DISCUSSION THREAD

**Discussion by MattiaLeoni** [verze-app/solana-php-sdk#37] [Q&A]
Hello, 
I used the function getTokenLargestAccounts to get the Token account (address), but from here i would like to get the owner wallet.
Is there a way I can accomplish that?

Thank you

**Comment by neverything** [comment]
Please refer to the official Solana docs: https://docs.solana.com/de/developing/clients/jsonrpc-api#gettokenlargestaccounts the response has the wallet addresses of the token accounts and they in turn should have the owner using https://docs.solana.com/de/developing/clients/jsonrpc-api#getaccountinfo iirc
